third_party/gpus/cuda_configure.bzl

```diff
@@ -209,6 +213,38 @@ def _enable_cuda(repository_ctx):
   return False
 
 
+def _cuda_nvcc_path(repository_ctx):
+  """Finds the cuda toolkit directory.
+
+  Args:
+    repository_ctx: The repository context.
+
+  Returns:
+    A speculative real path of the cuda toolkit install directory.
+  """
+  cuda_nvcc_path = _DEFAULT_CUDA_NVCC_PATH
+  if _CUDA_NVCC_PATH in repository_ctx.os.environ:
+    cuda_nvcc_path = repository_ctx.os.environ[_CUDA_NVCC_PATH].strip()
+  if not repository_ctx.path(cuda_nvcc_path).exists:
+    auto_configure_fail("Cannot find cuda nvcc path.")
```

Comment:
I think better error msg would be "Path %s does not exist" % cuda_nvcc_path or something like that. This error is unclear imo